src/main/java/net/atos/zac/documentcreatie/DocumentCreatieService.java

```diff
@@ -127,4 +128,18 @@ private Registratie createRegistratie(final DocumentCreatieGegevens documentCrea
         registratie.auditToelichting = AUDIT_TOELICHTING;
         return registratie;
     }
+
+    /**
+     * Lists all templates available for the current user
```

Comment:
`Lists all SmartDocuments templates available for the current user` I would suggest. :-) The more explicit, the better in my opinion.